src/Microsoft.DocAsCode.Build.SchemaDriven/Validators/SchemaValidator.cs

```diff
@@ -30,9 +30,23 @@ public static void Validate(DocumentSchema schema)
                 var isValid = o.IsValid(metaSchema, out IList<string> errors);
                 if (!isValid)
                 {
-                    throw new InvalidSchemaException($"Schema {schema.Title} is not a valid one according to {SupportedMetaSchemaUrl}: \n{errors.ToDelimitedString("\n")}");
+                    throw new InvalidSchemaException($"Schema {schema.Title} is not a valid one according to {SupportedMetaSchemaUri.OriginalString}: \n{errors.ToDelimitedString("\n")}");
                 }
             }
         }
+
+        public static bool ValidateSchemaUrl(string url)
+        {
+            if (url == null || !Uri.TryCreate(url, UriKind.Absolute, out Uri uri))
+            {
+                return false;
+            }
+            if (uri.Host == SupportedMetaSchemaUri.Host && uri.LocalPath == SupportedMetaSchemaUri.LocalPath && (string.IsNullOrEmpty(uri.Fragment) || uri.Fragment == "#"))
```

Comment:
seems host name ignore case

---
In reply to: [133377688](https://github.com/dotnet/docfx/pull/1962#discussion_r133377688) [](ancestors = 133377688)